sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/PartitionBasedLoadBalancer.java

```diff
@@ -317,8 +317,6 @@ private void renewOwnership(Map<String, PartitionOwnership> partitionOwnershipMa
             .subscribe(partitionPumpManager::verifyPartitionConnection,
                 ex -> {
                     LOGGER.error("Error renewing partition ownership", ex);
-                    ErrorContext errorContext = new ErrorContext(partitionAgnosticContext, ex);
-                    processError.accept(errorContext);
```

Comment:
Oh, it's my code. Sorry for that. And is there some documents about storage service side that help me know about their api including error code , etc. ?